Add bear app plugin

malexer opened the pull request:
Added Bear App plugin.

This plugin helps to integrate Sublime Text with [Bear App](http://www.bear-writer.com) for taking notes.
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks, depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass. 
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can trigger @packagecontrol-bot to re-evaluate your pull request
by pushing a commit or closing and reopening your pull request.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"` 
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.
 
You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists, 
why you believe it is different and needed
below this line. -->

FichteFoll commented:
@Thom1729 the package uses a custom URI scheme, not a HTTP request, so POST is out of the question.

Have you tried whether using `webbrowser.open` instead calling `osascript` also works? It does for me, on Linux.

Normally, just providing Command Palette entries (and maybe menu items) is enough. Ideally, the user should add key bindings by themselves as there are only so many bindings available, but the number of packages (with bindings) keeps growing.

malexer commented:
@Thom1729 as it was already mentioned, I am using [Custom URL Scheme](https://developer.apple.com/documentation/uikit/core_app/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app) to create a Note in Bear App.

I have tested the last version (webbrowser module) with 1 MB text - it worked, but Bear App was very slow consuming that data. In fact: Bear was also slow just to show that note.

@FichteFoll :
1. Thanks, forgot about that module. Fixed.
2. Absolutely valid, removed default key bindings.

Mentioned fixes were pushed to the repo as 1.0.1 version.
